config/prow/config.yaml

```diff
@@ -57,10 +57,10 @@ deck:
     - lens:
         name: buildlog
         config:
-          #highlighter:
-          #  endpoint: http://localhost:1234, http://some-service[.namespace.svc.cluster.local]
-          #  pin: true
-          #  auto: false
+          highlighter:
+            endpoint: http://halogen.default.svc.cluster.local
```

Comment:
FWIW I think you could just use `http://halogen`
e.g. https://github.com/kubernetes/test-infra/blob/1bca9bdedffc3fccad122e52b3c77dbd2324b770/config/prow/cluster/deck_deployment.yaml#L49

Reply:
Right, but I want the less ambiguous link